src/Aspire.Hosting/Utils/PasswordGenerator.cs

```diff
@@ -55,8 +55,8 @@ internal static class PasswordGenerator
     /// log base 2 [67^x * 23^a * 23^b * 10^c * 11^d * (a + b + c + d)! / (a! * b! * c! * d!)]
     /// </remarks>
     public static string Generate(int minLength,
-        bool lower, bool upper, bool numeric, bool special,
-        int minLower, int minUpper, int minNumeric, int minSpecial)
+        bool lower = true, bool upper = true, bool numeric = true, bool special = true,
+        int minLower = 0, int minUpper = 0, int minNumeric = 0, int minSpecial = 0)
```

Comment:
IMO don't add defaults. Just use named arguments in the few places this is called. Keeps it clear about what is generated.

Reply:
If nothing's actually using these arguments, and the type's not public, I'd consider removing them. Can always add them later.

Reply:
Reverted default args and added named args to callsite.